sdk/core/azure-core/src/base64.cpp

```diff
@@ -8,10 +8,10 @@
 
 namespace {
 
-static char const Base64EncodeArray[65]
```

Comment:
What's the rationale here to remove static?

Reply:
Not 100% sure if this is the logic from the tools that Anton is running, but declaring the type as "static" removes the symbol from the symbol table, which makes it impossible to address from outside the module.

But putting the type in an anonymous namespace has the identical effect, so it mostly just makes it harder to debug things.